common/utils/src/main/resources/error/error-conditions.json

```diff
@@ -2907,6 +2907,12 @@
     ],
     "sqlState" : "42613"
   },
+  "INVALID_REGEXP_REPLACE" : {
+    "message" : [
+      "Could not perform regexp_replace for `source = \"<source>\"`, `pattern = \"<pattern>\"`, `replacement = \"<replacement>\"` and `position = <position>`."
```

Comment:
Please, remove backticks. They are not necessary here, and we use them to highlight message parameters, literals and so on.

Reply:
```suggestion
      "Could not perform `regexp_replace` for source = \"<source>\", pattern = \"<pattern>\", replacement = \"<replacement>\" and position = <position>."
```